src/App.js

```diff
@@ -1,11 +1,24 @@
 import "./App.css";
+import react from "react";
+import {BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import Navbar from "./components/Navbar";
+import Home from "./components/Home";
+import Locations from "./components/Locations";
+import People from "./components/People";
+import Movies from "./components/Movies";
+
 
 function App() {
   return (
     <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
+      <Navbar />
+      <Switch>
+         <Route exact path='/' component={Home}/>
+         <Route exact path='/locations' component={Locations}/>
+         <Route exact path='/movies' component={Movies}/>
+         <Route exact path='/people' component={People}/>
+      </Switch>
```

Comment:
```suggestion
      <Switch>
         <Route exact path='/' component={Home}/>
         <Route path='/locations' component={Locations}/>
         <Route path='/movies' component={Movies}/>
         <Route path='/people' component={People}/>
      </Switch>
```
only the Home Route has to have an **exact** path. For the other three you can just put "path", because no other routes will partially match any of them.